x-pack/plugins/security_solution/public/management/components/artifact_entry_card/components/criteria_conditions.tsx

```diff
@@ -67,11 +81,30 @@ export const CriteriaConditions = memo<CriteriaConditionsProps>(
         .join(', ');
     }, [os]);
 
+    const getEntryValue = (type: string, value: string | string[]) => {
+      if (type === 'match_any' && Array.isArray(value)) {
+        return value.map((currentValue) => <EuiBadge color="hollow">{currentValue}</EuiBadge>);
+      }
+      return value;
+    };
+
+    const getEntryOperator = (type: string, operator: string) => {
+      if (type === 'nested') return;
+      return operator === 'included'
+        ? OPERATOR_TYPE_LABELS_INCLUDED[type as keyof typeof OPERATOR_TYPE_LABELS_INCLUDED] ?? type
+        : OPERATOR_TYPE_LABELS_EXCLUDED[type as keyof typeof OPERATOR_TYPE_LABELS_EXCLUDED] ?? type;
+    };
```

Comment:
Looks like these are pure and only used from within the `getNestedEntriesContent()` below. Maybe make them static functions in this module or move them to be inside of the `useCallback` for `getNestedEntriesContent()`?

Reply:
This is used in two sides. On the first level and for the nested conditions ( line 125 and line 155)